PublicationsRouter/v2/oai-pmh/VERBS.md

```diff
@@ -138,7 +181,36 @@ Returned information:
 
 * Record: notification metadata, only oai_dc metadata prefix is supported
 
-PubRouter API request:
+Example request: `GET https://pubrouter.jisc.ac.uk/oaipmh/repo/123456789?verb=GetRecord&identifier=987654321&metadataPrefix=oai_dc`
+
+Example return: 
+```xml
+<?xml version='1.0' encoding='UTF-8'?>
+<OAI-PMH xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.openarchives.org/OAI/2.0/" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
+    <responseDate>2018-05-23T14:29:54Z</responseDate>
+    <request verb="GetRecord" identifier="oai:pubrouter.jisc.ac.uk/notification:d2b949b769e1451da6792fa10b1c420b" metadataPrefix="oai_dc">http://pubrouter.jisc.ac.uk/repo/123456789</request>
+    <GetRecord>
+        <record>
+            <header xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/">
+                <identifier>oai:pubrouter.jisc.ac.uk/notification:987654321</identifier>
+                <datestamp>2018-02-01T15:21:06Z</datestamp>
+            </header>
+            <metadata xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/">
+            ... (for example of metadata go to XWALK.md)
+            </metadata>
+        </record>
+    </GetRecord>
+</OAI-PMH>
+```
+
+## ListSets (not yet properly supported)
 
-    GET /notification/<notification id>
+This request asks the OAI-PMH server to list the sets that are available to the client
```

Comment:
--> `This verb requests the OAI-PMH server to list the sets that are available to the client.  Note that sets are not supported by PubRouter, so use of this verb will result in an empty list response.`